src/backend/src/services/recruitment.services.ts

```diff
@@ -35,4 +35,11 @@ export default class RecruitmentServices {
 
     return milestone;
   }
+
```

Comment:
Add javadoc comments. also add them for the endpoint above I forgot to to do that in mine